config/initializers/new_framework_defaults_5_1.rb

```diff
@@ -0,0 +1,14 @@
+# Be sure to restart your server when you modify this file.
+#
+# This file contains migration options to ease your Rails 5.1 upgrade.
+#
+# Once upgraded flip defaults one by one to migrate to the new default.
+#
+# Read the Guide for Upgrading Ruby on Rails for more info on each option.
+
+# Make `form_with` generate non-remote forms.
+Rails.application.config.action_view.form_with_generates_remote_forms = false
```

Comment:
If you plan to keep this setting, we need to move it into a config file. This `config/initializers/new_framework_defaults_5_1.rb` is a temporary file for use to help us turn on/off new settings for a slow upgrade rollout. We should plan to delete this file soon.

Reply:
Thanks for insights. I had no idea! After reading up on it, it looks like it only impacts `form_with` (which we're not currently using) and not `form_for`.

If we get to a point where we add a new form, surely we'll check to see what it's doing and can address it then?

So I think this can just be skipped.

The only other line in this file is commented out, so I think I'll just remove this whole file?

Reply:
Yep! Sounds good to me.